web/src/js/components/Modes.tsx

```diff
@@ -27,7 +30,10 @@ export default function Modes() {
                     ) : (
```

Comment:
Remove the check above and check whether localModeUnavailable is set instead?

Reply:
Adjusted! The `platform` doesn't disappear, though, since we need it for **transparent mode**.